clients/admin-ui/cypress/e2e/systems.cy.ts

```diff
@@ -618,23 +618,35 @@ describe("System management page", () => {
 
     describe("delete privacy declaration", () => {
       beforeEach(() => {
-        cy.fixture("systems/systems_with_data_uses.json").then((systems) => {
+        cy.fixture("systems/system.json").then((system) => {
           cy.intercept("GET", "/api/v1/system/*", {
-            body: systems[0],
-          }).as("getFidesctlSystemWithDataUses");
+            body: system,
+          }).as("getDemoAnalyticsSystem");
         });
-        cy.visit(`${SYSTEM_ROUTE}/configure/fidesctl_system`);
-        cy.wait("@getFidesctlSystemWithDataUses");
+        cy.visit(`/${SYSTEM_ROUTE}/configure/demo_analytics_system`);
+        cy.wait("@getDemoAnalyticsSystem");
         cy.fixture("systems/system.json").then((system) => {
-          const newSystem = { ...system, fides_key: "fidesctl_system" };
+          const newSystem = { ...system, fides_key: "demo_analytics_system" };
           cy.intercept("PUT", "/api/v1/system*", { body: newSystem }).as(
-            "putFidesSystem"
+            "putDemoAnalyticsSystem"
           );
         });
 
         cy.getByTestId("tab-Data uses").click();
       });
 
+      it("can visit the privacy declaration tab", () => {
```

Comment:
nit: it feels a little weird for this test to be under the "delete privacy declaration" section